Desktop/qquick/rcommander.cpp

```diff
@@ -63,15 +63,13 @@ bool RCommander::addAnalysis(const QString &code)
 
 	setRunning(true);
 
-	// Temporary solution: check whether the code starts with '<moduleName>::<analysisNameWrapper>(...'
+	// Temporary solution: check whether the code starts with '<moduleName>::<analysisName>(...'
 	QString codeTrimmed = code.trimmed();
 	QStringList analysisParts = codeTrimmed.mid(0, codeTrimmed.indexOf("(")).split("::");
 
 	if (analysisParts.length() == 2)
```

Comment:
Instead of doing all this you could also simply make a `std::map<std::string....` that maps all the known "wrappers" or "syntax mode calls" to `Analysis` but that isnt quite necessary in this PR

Reply:
I have made the parsing a bit more clever so that it checked that the moduleName & analysisName exist